node/lib/util/commit.js

```diff
@@ -497,7 +498,8 @@ exports.commit = co.wrap(function *(metaRepo,
                                     metaStatus,
                                     message,
                                     subMessages,
-                                    noVerify) {
+                                    noVerify,
+                                    mergeParent) {
```

Comment:
could you also update the doc with the new parameter, and and a short description on what happens if `mergeParent` is present?